src/ImageSharp/Formats/Png/PngDecoderCore.cs

```diff
@@ -380,7 +380,12 @@ private void ReadGammaChunk(PngMetadata pngMetadata, ReadOnlySpan<byte> data)
         private void InitializeImage<TPixel>(ImageMetadata metadata, out Image<TPixel> image)
             where TPixel : unmanaged, IPixel<TPixel>
         {
-            image = new Image<TPixel>(this.configuration, this.header.Width, this.header.Height, metadata);
+            image = Image.CreateUninitialized<TPixel>(
```

Comment:
Easy win on decode here. This was taking a larger percentage of decoding time than I expected.